tests/boards/tenstorrent/tt_blackhole/dmc/gpiox/src/main.c

```diff
@@ -13,21 +13,26 @@
 #define NGPIOS 8
 
 static const struct device *const ports[] = {
-	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox0)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox1)),
-	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox2)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox3)),
-	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox4)), DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox5)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox2)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox3)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox4)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(gpiox5)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(chip0_strapping)),
+	DEVICE_DT_GET_OR_NULL(DT_NODELABEL(chip1_strapping)),
 };
 
 /* clang-format off */
 static const bool expected_availability[] = {
 #ifdef CONFIG_BOARD_REVISION_P100
-	true, false, true, true, true, true
+	true, true, true, true, false, false,
```

Comment:
It probably makes sense to add to this list rather than remove from it.
```suggestion
	true, false, true, true, true, true, false, false,
```